app/initializers/responsive.js

```diff
@@ -1,2 +1,32 @@
-import responsive from 'ember-responsive/initializers/responsive';
-export default responsive;
+import Ember from 'ember';
+import config from '../config/environment';
```

Comment:
If you wanted to keep this initializer in `addon/`, you could also consider [ember-get-config](https://github.com/null-null-null/ember-get-config).